src/main/java/io/kestra/plugin/gitlab/core/Create.java

```diff
@@ -1,5 +1,6 @@
-package io.kestra.plugin.gitlab.issues;
+package io.kestra.plugin.gitlab.core.issues;
```

Comment:
No... You have to keep the tasks that were in the issues package...

The target tasks to move in `io.kestra.plugin.gitlab.core` are ONLY the one in the root package `io.kestra.plugin.gitlab`.